docs/tutorial/common_hpc_dependencies.rst

```diff
@@ -86,6 +86,11 @@ Test. ``test_2.cpp`` provides an example driver for MPI with GoogleTest.
   BLT also has the variable ``ENABLE_FIND_MPI`` which turns off all CMake's ``FindMPI``
   logic and then uses the MPI wrapper directly when you provide them as the default
   compilers.
+  
+  BLT also provides ``BLT_USE_FIND_MPI_TARGETS`` that will directly use the
+  ``MPI::MPI_C`` and ``MPI::MPI_CXX`` targets provided by CMake's ``FindMPI``.
```

Comment:
Please add that it those targets are able to be referenced under the unified name of `mpi`

Reply:
done

Reply:
two backticks

Reply:
two backticks times two.